### PR TITLE
Document missing bundled conda for self hosted runners

### LINKED ISSUE #113: Error: Bundled Miniconda not found!
hi! congrats for this package - it's really helpful.

we are using it in [`kornia`](https://github.com/arraiyopensource/kornia) for building our CI pipeline and we are facing some issues with the setup for a self-hosted device.

In particular we have the following error:

image
https://github.com/kornia/kornia/runs/1448438566?check_suite_focus=true

The `.yml` used to define this workflow can be found here:
https://github.com/kornia/kornia/blob/master/.github/workflows/tests_cuda.yml

Thanks in advance,

### PR DESCRIPTION
**Pull request by jaimergp**
Closes #113 

---

I think documenting the workarounds in the error message is enough for this corner case.

### CODE REVIEW

File: src/setup.ts [hunk 1]
```diff
@@ -48,7 +48,11 @@ async function setupMiniconda(inputs: types.IActionInputs): Promise<void> {
   }
 
   if (!fs.existsSync(basePath)) {
-    throw Error(`No installed conda 'base' enviroment found at ${basePath}`);
+    throw Error(`No installed conda 'base' enviroment found at ${basePath}!` +
+                'If you are using this action in a self-hosted runner that already provides ' +
+                'its own Miniconda installation, please specify its location with a `CONDA` ' +
+                'environment variable. If you want us to download and install Miniconda for ' +
+                'you, use `miniconda-version: "latest"` in the parameters for this action.');
```
Comment (bollwyvl) [inline src/setup.ts:55]: We might also want to rep `miniforge-version`...

Comment (goanpeca) [reply src/setup.ts:55]: @jaimergp ?

Comment (jaimergp) [reply src/setup.ts:55]: Oops missed this, pushing now.

File: src/setup.ts [hunk 2]
```diff
@@ -48,7 +48,12 @@ async function setupMiniconda(inputs: types.IActionInputs): Promise<void> {
   }
 
   if (!fs.existsSync(basePath)) {
-    throw Error(`No installed conda 'base' enviroment found at ${basePath}`);
+    throw Error(`No installed conda 'base' enviroment found at ${basePath}!` +
```
Comment (aselius) [inline src/setup.ts:51]: type s/enviroment/environment 😅

Comment (jaimergp) [reply src/setup.ts]: Thanks!

**Comment by jaimergp** [comment]
@goanpeca this should be ready to go

**Comment by traversaro** [comment]
fyi @singhbal-baljinder

**Pull request merged**